lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -177,6 +179,95 @@ sub run_cromwell {
     $self->_stage_cromwell_outputs($results_dir);
 }
 
+sub run_cromwell_gcp {
+    my $self = shift;
+    my $yaml = shift;
+    my $results_dir = shift;
+
+    my $zip_deps = Genome::Config::get('workflow_deps_zip');
```

Comment:
It can be generated from the analysis-wdls directory that it contains. Outside of GMS I do that on submit
https://github.com/griffithlab/cloud-workflows/blob/main/scripts/submit_workflow.sh#L66-L76